src/dialects/abstract/index.ts

```diff
@@ -42,7 +42,10 @@ export type DialectSupports = {
   },
   /* Do we need to say DEFAULT for bulk insert */
   bulkDefault: boolean,
+  /** Whether this dialect has native support for schemas */
   schemas: boolean,
+  /** Whether this dialect has native support for databases (in the postgres or mssql sense) */
```

Comment:
I think it will be clearer if I rename the property to `multiDatabases`.

They work similarly to schemas. You can think of them as a group of schemas, where schemas are a group of tables.

Some dialects support them, others do not. If I display all my locally running test databases, you can easily see which ones support what:

![image png-mh](https://user-images.githubusercontent.com/1280915/190988736-9c4e9d23-0ecc-4eb1-bf26-29a6bdcc1e55.png)

MariaDB not listed but works the same as MySQL.

Basically only postgres & mssql support having multiple databases, but all of them support schemas.

MySQL decided to make `CREATE DATABASE` and `CREATE SCHEMA` synonymous, but for our purposes, they behave like schemas (a collection of tables). MySQL also calls them schemas in their system tables, even though their documentation refers to them as databases.

Fixing this distinction is also a good step towards fixing https://github.com/sequelize/sequelize/issues/12449

Reply:
I agree that 'multiDatabases' is a clearer distinction. The image makes it very clear, thanks for that explanation!

Reply:
Done